src/Dapper.FluentMap/TypeMaps/MultiTypeMap.cs

```diff
@@ -104,7 +104,7 @@ public SqlMapper.IMemberMap GetMember(string columnName)
                     if (result != null)
                     {
 #if !NETSTANDARD1_3
-                        if (result is IgnoredPropertyInfo)
+                        if (result is IgnoredPropertyInfo || result.Property is IgnoredPropertyInfo)
```

Comment:
I think ```result``` will always be an ```IMemberMap```. Only the property can be ```IgnoredPropertyInfo```
```suggestion
                        if (result.Property is IgnoredPropertyInfo)
```